addons/rook/template/base/install.sh

```diff
@@ -988,3 +994,16 @@ function rook_prompt_migrate_from_longhorn() {
         bail "Not migrating"
     fi
 }
+
+# wait for Rook deployment to be running and healthy
+function rook_maybe_wait_for_rollout() {
+    log "Awaiting Rook pods to transition to Running"
+    if ! spinner_until 120 wait_for_running_pods "rook-ceph"; then
+        logWarn "Rook-ceph rollout did not complete"
+    fi
+
+    log "Awaiting for Ceph Cluster to be Ready"
+    if ! spinner_until 120 rook_cephcluster_ready; then
```

Comment:
does it make sense to wait for ready before pods? the operator may restart pods if its still updating.

Reply:
addressed in #4101